docs/guides/data-movement/readme.md

```diff
@@ -90,6 +90,8 @@ The `CreateRequest` API call that is used to create Data Movement with the Copy
 options to allow a user to specify some options for that particular Data Movement. These settings
 are on a per-request basis.
 
+The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may be configured to run full-time, or it may left in a disabled state if the WLM is expected to run it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd service configuration of the daemon. See `RequiredDaemons` in [Directive Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the daemon, in the case where the WLM will run it only on demand.
+
```

Comment:
Can you also add something about having to use `--requires=copy-offload` in the DW directive?

Reply:
hmm, I wanted to be careful about documenting the same thing in multiple places.  I think the reference to the other doc should cover it.

Reply:
Understood. But I don't think a user that wants to use the Copy Offload API is going to care much about DirectiveBreakdowns.

Reply:
I see your point. Hang on.